code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -80,6 +80,7 @@ var/global/list/datum/stack_recipe/metal_recipes = list (
 		)),
 	null,
 	new/datum/stack_recipe("iron door", /obj/machinery/door/mineral/iron, 20, one_per_turf = 1, on_floor = 1),
+	new/datum/stack_recipe("stove", /obj/machinery/space_heater/campfire/stove, 5, time = 25, one_per_turf = 1, on_floor = 1),
```

Comment:
You should not be able to make this straight out of metal, otherwise you'll see hundreds of stoves inside the station every round